jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -273,6 +274,14 @@ class DeploymentFlow extends React.Component {
     this.setState({region: region});
   }
 
+  /**
+    Store the selected SLA details.
+    @param {Object} sla The selected sla information.
```

Comment:
Is this really an object? Not a string?
If it is an object, what type of object?

Reply:
Yes, it's an object because it needs to contain, not only the name of the sla but also it's costs and machine counts etc. I'll expand the comment.